### PR TITLE
create channel script

### PR DESCRIPTION
**Pull request by IslandRhythms**

### CODE REVIEW

File: createChannel.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 const axios = require('axios');
 const {WebClient} = require('@slack/web-api');
-const config = require('./.config/.config.js');
+const config = require('./.config/development.js')
```
Comment (vkarpov15) [inline createChannel.js]: Undo this please. Should be `require('./.config')`, and run with `env NODE_ENV=development` if you want to use development.js

File: createChannel.js [hunk 2]
```diff
@@ -11,10 +11,12 @@ const r1 = readline.createInterface({
...
 
 async function run() {
-    const client = new WebClient(config.userSlackToken); // Use the User OAuth Token, not Bot User Oauth Token
+    const client = new WebClient(config.slackToken);
     let res = await client.conversations.list({types: 'public_channel, private_channel'});
     res = res.channels.map((item) => item.name_normalized);
     console.log(res);
+    const val = await client.users.lookupByEmail({email: 'val@karpov.io'});
+    const daniel = await client.users.lookupByEmail({email: 'daniel@meanitsoftware.com'});
     r1.question('What is the name of the channel you wish to create? ', function(name) {
```
Comment (vkarpov15) [inline createChannel.js:19]: Fine for now, but in the future please no callbacks :)

**Review by vkarpov15 — CHANGES_REQUESTED** [review]

File: createChannel.js (hunk 2 above)
Comment (IslandRhythms) [reply createChannel.js:19]: Didn't realize it at the time but its due to the version of node. v 17 you can save the result to a variable but before that it has to be this.

**Review by vkarpov15 — APPROVED** [review]
Thanks :+1:

**Pull request merged**